gemrb/core/Cache.h

```diff
@@ -32,6 +34,13 @@ namespace GemRB {
 typedef void (*ReleaseFun)(void *);
 #endif
 
+typedef std::array<char, 9> ieResRefKey;
```

Comment:
let's not use `typedef`. now that we use C++11 `using` is the way.

moot for this, since we ought to just use the `ResRef` class here too.